testing/functional/tests/cypress/integration/student/04-application-page.spec.ts

```diff
@@ -0,0 +1,211 @@
+    applicationObject
+      .startApplicationStudyYearDisableButton()
+      .should("be.visible");
+    applicationObject.selectStudyYearDropdown();
+    applicationObject
+      .startApplicationStudyYearEnableButton()
+      .should("not.exist");
+  });
+
+  it("Verify that Start Application button must be disable if selected study year is removed in application form.", () => {
+    applicationObject.applicationButton().should("be.visible").click();
+    applicationObject.startNewApplicationButton().should("be.visible").click();
+    applicationObject.selectStudyYearDropdown();
+    applicationObject
+      .startApplicationStudyYearEnableButton()
+      .should("not.exist");
+    applicationObject.removedButton().click();
+    applicationObject
+      .startApplicationStudyYearDisableButton()
+      .should("be.visible");
+  });
+
+  it("Verify that Start Application button must be clickable in application form.", () => {
+    applicationObject.applicationButton().should("be.visible").click();
+    applicationObject.startNewApplicationButton().should("be.visible").click();
+    applicationObject
+      .startApplicationStudyYearDisableButton()
+      .should("be.visible");
+    applicationObject.selectStudyYearDropdown();
+    applicationObject
+      .startApplicationStudyYearEnableButton()
+      .should("not.exist");
+
+    //Info:- On click -> Alert in progress popup open
+    applicationObject.startApplicationTuitionWaiverButton().click();
+    applicationObject.applicationAlreadyInProgressText().should("be.visible");
+    applicationObject.applicationAlreadyInProgressTextCard().click();
+
+    applicationObject.startApplicationBCloanForgivenessProgramButton();
+    applicationObject.applicationAlreadyInProgressText().should("be.visible");
+    applicationObject.applicationAlreadyInProgressTextCard().click();
+
+    applicationObject.startApplicationPacificLeaderLoanForgivenessButton();
+    applicationObject.applicationAlreadyInProgressText().should("be.visible");
+    applicationObject.applicationAlreadyInProgressTextCard().click();
+
+    applicationObject.startApplicationInterestFreeStatusButton();
+    applicationObject.applicationAlreadyInProgressText().should("be.visible");
+    applicationObject.applicationAlreadyInProgressTextCard().click();
+  });
+
+  it("By clicking on edit button it redirects to Welcome Page in application form.", () => {
+    applicationObject.applicationButton().should("be.visible").click();
+    applicationObject.draftApplication().click();
+    applicationObject.draftApplicationVerifyText().should("be.visible");
+  });
+
+  it("By clicking on Next Section button from Welcome Page it redirects to Program Page in application form.", () => {
+    applicationObject.applicationButton().should("be.visible").click();
+    applicationObject.draftApplication().click();
+    applicationObject.draftApplicationVerifyText().should("be.visible");
+    applicationObject.NextSectionButton();
```

Comment:
Why do we need a navigate method at the end of the test if no other assertion is happening? Maybe I am missing something, can you please clarify?